app/assets/javascripts/school_administrator_dashboard/page_container.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import _ from 'lodash';
+import SchoolAbsenceDashboard from './school_absence_dashboard.jsx';
+
+export default React.createClass({
+  displayName: 'DashboardPageContainer',
+
+  propTypes: {
+    attendanceData: React.PropTypes.object.isRequired  //should rename more generically
```

Comment:
@edavidsonsawyer I see a TODO here, is this something you want to do before or after merging?